crates/e2e-testing/src/spin_controller.rs

```diff
@@ -112,9 +120,16 @@ impl Controller for SpinUp {
 
     async fn stop_app(
         &self,
-        _: Option<&str>,
+        appname: Option<&str>,
         process: Option<tokio::process::Child>,
     ) -> Result<()> {
+        if spin::cleanup_state_dir(appname.unwrap()).is_err() {
```

Comment:
This is very pernickety, but...  my personal taste would be to delete the state directory only once the app had stopped running.  In practice, though, it's not going to be an issue given our usage patterns.

Reply:
this is such a silly mistake, I am feeling ashamed. fixed :)